Library/Homebrew/gpg.rb

```diff
@@ -0,0 +1,47 @@
+require "utils"
+
+class Gpg
+  # Should ideally be using `GPGRequirement.new.gpg2`, etc to get path here but
+  # calling that directly leads to:
+  # requirement.rb:139:in `which_all': uninitialized constant Requirement::ORIGINAL_PATHS (NameError)
```

Comment:
I thought about that. Wasn't sure if it was acceptable or not, but if it is, sounds reasonable.

I had some issues around `which_all` always returning a `Pathname` even if you forced `to_s` on it but I think I've just worked out why that wasn't working. Will double check with a fuller test locally.